### DISCUSSION TITLE
Use off the shelf hardware to make the project easier to reproduce

### DISCUSSION THREAD

**Discussion by MarcSN311** [OMOTE-Community/OMOTE-Firmware#52] [Ideas]
Hi!

I wanted to build something like this for years but never came around to do it. Great Project!

I was thinking about using of the shelf hardware to make it cheaper and more accessible (easier to build). I found this:
https://www.aliexpress.com/item/1005005138982767.html
Which basically ony needs some buttons and IR LEDs. Sadly the screen is 3.2" not 2.8" so the case would need to be changed. There is also a 2.4" variant. 2.8"is available but only with resistive touchscreen.

**Comment by Dark1886** [comment]
I don't see any reason you wouldn't be able to use this screen, although you may be stuck with weird looking LVGL items bevuase they were written for a smaller screen. 

If you did want to use something like this, just make sure the screen is ran by the same or a similar chip and you have the pinout so you can change the settings in the current project code.
